lib/graphics.py

```diff
@@ -2021,6 +2029,11 @@ def get_sprite_at_position(self, x, y):
 
 
     def __check_mouse(self, x, y):
+        if self.scale:
+            ax, ay = self._get_aspect_x_y()
+            x /= ax
```

Comment:
just a style knit - could you rephrase this as ```x = x / ax``` - don't have strong reasons but i think helps clarity